From 6b6e0552cdfbacc864e87bf007c718786f56da38 Mon Sep 17 00:00:00 2001
From: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
Date: Tue, 1 Sep 2026 15:31:06 +0200
Subject: [PATCH 32/38] ipa: rppx1: imx708: Add CCM tuning

Add CCM values for a fixed set of colour temperatures.

The values are identical from the ones available in the RaspberryPi
tuning file for imx708.

Signed-off-by: Jacopo Mondi <jacopo.mondi@ideasonboard.com>

Upstream-Status: Pending

---
 src/ipa/rppx1/data/imx708.yaml | 31 +++++++++++++++++++++++++++++++
 1 file changed, 31 insertions(+)

diff --git a/src/ipa/rppx1/data/imx708.yaml b/src/ipa/rppx1/data/imx708.yaml
index 3057df83..3fcd3c56 100644
--- a/src/ipa/rppx1/data/imx708.yaml
+++ b/src/ipa/rppx1/data/imx708.yaml
@@ -7,5 +7,36 @@ algorithms:
   - Awb:
   - BlackLevelCorrection:
   - Ccm:
+      ccms:
+        - ct: 2964
+          ccm: [
+                 1.72129, -0.45961, -0.26169,
+                 -0.30042, 1.56924, -0.26882,
+                 0.15133, -1.13293, 1.98161
+               ]
+        - ct: 3610
+          ccm: [
+                 1.54474, -0.35082, -0.19391,
+                 -0.36989, 1.67926, -0.30936,
+                 -0.00524, -0.55197, 1.55722
+               ]
+        - ct: 4640
+          ccm: [
+                 1.52972, -0.35168, -0.17804,
+                 -0.28309, 1.67098, -0.38788,
+                 0.01695, -0.57209, 1.55515
+               ]
+        - ct: 5910
+          ccm: [
+                 1.56879, -0.42159, -0.14719,
+                 -0.27275, 1.59354, -0.32079,
+                 -0.02862, -0.40662, 1.43525
+               ]
+        - ct: 7590
+          ccm: [
+                 1.41424, -0.21092, -0.20332,
+                 -0.17646, 1.71734, -0.54087,
+                 0.01297, -0.63111, 1.61814
+               ]
   - GammaOutCorrection:
 ...
